src/actions/nodes.js

```diff
@@ -19,7 +19,7 @@ export const TOGGLE_NODES_DISABLED = 'TOGGLE_NODES_DISABLED';
 /**
  * Toggle a selected group of nodes' visibility on/off
  * @param {Array} nodeIDs The nodes' unique identifiers
- * @param {Boolean} isDisabled Whether the node should be visible
+ * @param {boolean} isDisabled Whether the node should be visible
```

Comment:
Per #1108 I think this should be `Boolean`.

Reply:
Ah, right!

```suggestion
 * @param {Boolean} isDisabled Whether the node should be visible
```

I'll not standardize in this PR, since @nikotome is already doing it in #1109